docs/components/encoder/_index.md

```diff
@@ -44,6 +44,262 @@ For configuration information, click on one of the following models:
+{{% tab name="Python" %}}
+
+```python
+from viam.components.encoder import Encoder
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+```go
+import (
+  "go.viam.com/rdk/components/encoder"
+)
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## API
+
+The encoder component supports the following methods:
+
+Method Name | Description
+----------- | -----------
+[GetPosition](#getposition) | Get the current position of the encoder.
+[ResetPosition](#resetposition) | Reset the position to zero.
+[GetProperties](#getproperties) | Get the supported properties of this encoder.
+[DoCommand](#docommand) | Send or receive model-specific commands.
+
+### GetPosition
+
+Get the current position of the encoder in ticks or degrees.
+Relative encoders return ticks since last zeroing.
+Absolute encoders return degrees.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `position_type` (PositionType.ValueType): Specify whether to get the current position in ticks (`encoder.PositionType.TICKS`) or in degrees (`encoder.PositionType.DEGREES`).
+  If you are not sure which position type your encoder supports but it is a built-in Viam-supported model, you can leave this parameter unspecified (`encoder.PositionType.UNSPECIFIED`) and it will default to the correct position type.
+- `extra` [(Optional[Dict[str, Any]])](https://docs.python.org/library/typing.html#typing.Optional): Extra options to pass to the underlying RPC call.
+- `timeout` [(Optional[float])](https://docs.python.org/library/typing.html#typing.Optional): An option to set how long to wait (in seconds) before calling a time-out and closing the underlying RPC call.
+
+**Returns:**
+
+- (Tuple[[float](https://docs.python.org/3/library/functions.html#float), PositionType.ValueType]): The current position in ticks or degrees, and the type of position the encoder returns (ticks or degrees).
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/components/encoder/client/index.html#viam.components.encoder.client.EncoderClient.get_position).
+
+```python
+my_encoder = Encoder.from_robot(robot=robot, name='my_encoder')
+
+# Get the position of the encoder in ticks
+position = await my_encoder.get_position(encoder.PositionType.TICKS)
+print("The encoder position is currently ", position[0], position[1])
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `ctx` [(Context)](https://pkg.go.dev/context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+- `positionType` ([PositionType](https://pkg.go.dev/go.viam.com/rdk/components/encoder#PositionType)): Specify whether to get the current position in ticks (`encoder.PositionTypeTicks`) or in degrees (`encoder.PositionTypeDegrees`).
+  If you are not sure which position type your encoder supports but it is a built-in Viam-supported model, you can leave this parameter unspecified (`encoder.PositionTypeUnspecified`) and it will default to the correct position type.
```

Comment:
I'm not sure why, but it seems like with Go you do have to pass in `encoder.PositionTypeUnspecified`